pkg/devfile/adapters/common/mocks/ComponentAdapter.go

```diff
@@ -0,0 +1,191 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+
+package mocks
```

Comment:
Generated mocks for nested interface too using `mockery`. I found ease of generating mocks for such kind of interfaces using mockery. However I will try to add mocks for such scenario with gomock as well which is little challenging and still working on that.